Formula/g/goimports.rb

```diff
@@ -0,0 +1,33 @@
+class Goimports < Formula
+  desc "Go formatter that additionally inserts import statements"
+  homepage "https://pkg.go.dev/golang.org/x/tools/cmd/goimports"
+  url "https://github.com/golang/tools/archive/refs/tags/v0.31.0.tar.gz"
+  sha256 "e5d74f1e63a1ee669e75e76668cea1b110e2b9d19c67710f60939ee38070a5a7"
+  license "BSD-3-Clause"
+  head "https://github.com/golang/tools.git", branch: "master"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "go"
+
+  def install
+    chdir "cmd/goimports" do
+      system "go", "build", *std_go_args
+    end
+  end
+
+  test do
+    (testpath/"main.go").write <<~EOS
+      package main
+
+      func main() {
+        fmt.Println("hello")
+      }
+    EOS
```

Comment:
A bit more nitpicking: 😅 
```suggestion
    (testpath/"main.go").write <<~GO
      package main

      func main() {
        fmt.Println("hello")
      }
    GO
```